src/components/TodoList/todolist.scss

```diff
@@ -0,0 +1,29 @@
+.todo-list {
+  width: 500px;
+  background-color: hsl(237deg, 14%, 26%);

```

Comment:
We can start to use more variables from SCSS, the reason is that I have no idea what is this color based on the HSL value.